tools/StaticAnalysis/Exceptions/Az.Storage/SignatureIssues.csv

```diff
@@ -56,8 +56,11 @@
 "Microsoft.Azure.PowerShell.Cmdlets.Storage.dll","Microsoft.WindowsAzure.Commands.Storage.Blob.Cmdlet.NewAzureStorageContainerSasTokenCommand","New-AzStorageContainerSASToken","1","8100","New-AzStorageContainerSASToken Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
 "Microsoft.Azure.PowerShell.Cmdlets.Storage.dll","Microsoft.WindowsAzure.Commands.Storage.Blob.Cmdlet.NewAzureStorageContainerSasTokenCommand","New-AzStorageContainerSASToken","1","8510","Cmdlet 'New-AzStorageContainerSASToken' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
 "Microsoft.Azure.PowerShell.Cmdlets.Storage.dll","Microsoft.WindowsAzure.Commands.Storage.Blob.Cmdlet.NewAzureStorageContainerStoredAccessPolicyCommand","New-AzStorageContainerStoredAccessPolicy","1","8100","New-AzStorageContainerStoredAccessPolicy Does not support ShouldProcess but the cmdlet verb New indicates that it should.","Determine if the cmdlet should implement ShouldProcess and if so determine if it should implement Force / ShouldContinue"
+"c:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\Az.Storage\Microsoft.Azure.PowerShell.Cmdlets.Management.Storage.dll","Microsoft.Azure.Commands.Management.Storage.GetAzureStorageAccountCommand","Get-AzStorageAccount","1","8410","Parameter IncludeGeoReplicationStats of cmdlet Get-AzStorageAccount does not follow the enforced naming convention of using a singular noun for a parameter name.","Consider using a singular noun for the parameter name."
+"c:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\Az.Storage\Microsoft.Azure.PowerShell.Cmdlets.Management.Storage.dll","Microsoft.Azure.Commands.Management.Storage.RemoveAzureStorageAccountManagementPolicyCommand","Remove-AzStorageAccountManagementPolicy","1","8600","Cmdlet 'Remove-AzStorageAccountManagementPolicy' has no defined output type.","Add an OutputType attribute that declares the type of the object(s) returned by this cmdlet. If this cmdlet returns no output, please set the output type to 'bool' and make sure to implement the 'PassThru' parameter."
```

Comment:
Please remove **c:\workspace\powershell\src\Package\Debug\ResourceManager\AzureResourceManager\Az.Storage**

Reply:
This is from maddieclayton  in commit https://github.com/Azure/azure-powershell/commit/c244fd27f880b82b1e300280f5ae99f8d378b757